### ISSUE TITLE
Terminals lose colors after wake from sleep

### ISSUE THREAD

**Issue by bst82551** [Eugeny/tabby#8763]
Version: 1.0.197
Platform: Linux x64 6.3.0-kali1-amd64
Plugins: none
Frontend: xterm-webgl

Upon waking from sleep, my terminals all lose their colors and go grey. I can still type text into them and interact with programs that were running (i.e. vim), but I can't see anything other than the cursor. The only way to fix it is close the terminal and reopen a new one.

**Comment by bst82551** [comment]
Here's a screenshot of the problem. I have 2 terminals open in split view, each running vim. Right pane even shows highlights from a text search, but fails to show the text.

![Screenshot_2023-08-03_07-09-34](https://github.com/Eugeny/tabby/assets/90732043/730eb59c-2311-466b-93c1-ce342a819990)

**Comment by InToSSH** [comment]
+1 I have the same issue, running Ubuntu 23.10, but it was happening on Pop OS 22.04 aswell. The result is the same using both webgl and canvas. However WebGL usually shows white empty screen.

**Comment by aqueos** [comment]
yes i have the same on tabby  1.0.207 on ubuntu 22.04 LTS after ssupend the tabby screnn is all a little smiley :-(
and all is gone.
